.github/actions/build_info/action.yml

```diff
@@ -26,10 +18,22 @@ outputs:
 runs:
   using: composite
   steps:
-    - name: Set Python version vars
+    - name: Configure Python version info
       id: build_info
-      run: ./.github/scripts/build_info.py
-      env:
-        GITHUB_CONTEXT: ${{ toJson(github) }}
-        GITHUB_INPUTS: ${{ toJson(inputs) }}
       shell: bash
+      run: |
+        PYTHON_MIN_VERSION="3.9"
+        PYTHON_MAX_VERSION="3.13"
+        PYTHON_VERSIONS='["min", "3.10", "3.11", "3.12", "max"]'
```

Comment:
It would be nice at some point that we automatically get min/max from the streamlit setup.py. However, the current version also requires manual configuration, and its not too bad since this is already a centralized place shared across all workflows and changing version support does not happen often.